src/components/call/CallGeneralInfo.tsx

```diff
@@ -90,9 +167,77 @@ const CallGeneralInfo: React.FC<{
         />
         <Field
           name="referenceNumberFormat"
+          validate={validateRefnumber}
           label="Reference number format"
           type="text"
           component={TextField}
+          InputProps={{
+            endAdornment: (
+              <InputAdornment position="end">
+                <IconButton onClick={handleClickOpen}>
+                  <HelpIcon />
+                </IconButton>
+                <Dialog
+                  onClose={handleClose}
+                  aria-labelledby="customized-dialog-title"
+                  open={open}
+                  color="primary"
+                >
+                  <DialogContent dividers>
+                    <Typography gutterBottom color="inherit" variant="body1">
+                      A reference number format determines how reference numbers
```

Comment:
You could put this text in its own variable to keep the render() method a little bit tidier. Similarly you could also pull out the table (which is quite verbose) into a component, though I'm not sure how necessary that is